controllers/profiles/dev/object_creators_dev.go

```diff
@@ -139,7 +138,7 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *corev1.ConfigMap, workf
 				}
 				// the resource configMap needs a specific dir, inside the src/main/resources
 				// to avoid clashing with other configMaps trying to mount on the same dir, we create one projected per path
-				volumeMountName := configMapExternalResourcesVolumeNamePrefix + utils.PathToString(workflowResCM.WorkflowPath)
+				volumeMountName := kubeutil.MustSafeDNS1035(configMapExternalResourcesVolumeNamePrefix, workflowResCM.WorkflowPath)
```

Comment:
sorry for the silly question, mostly becasue I'm not yet fully familiar with go..., but in the case of a wrong name, the exectuion of MustSafeDNS1035 might lead in a panic(xxxx) invocation. In that case, what happens with all this execution path? I mean, when the panic exectuion starts to "kill" the exectuion chain, what are is being killed at head of that chain. A reconcile thread? the controller?

Reply:
It will kill the pod.